lib/utils/error-messages.js

```diff
@@ -200,16 +201,20 @@ function getPropertyNotAllowedErrorMessage(report, executionPlatformLabel, model
     property
   } = error;
 
-  if (property === 'modelerTemplate') {
-    const typeString = getTypeString(parentNode || node);
+  const typeString = getTypeString(parentNode || node);
 
+  if (property === 'modelerTemplate') {
     if (modeler === 'desktop') {
       return `${ getIndefiniteArticle(typeString) } <Template ${ typeString }> is not supported by ${ executionPlatformLabel }`;
     } else if (modeler === 'web') {
       return `${ getIndefiniteArticle(typeString) } <Connector ${ typeString }> is not supported by ${ executionPlatformLabel }`;
     }
   }
 
+  if (is(node, 'bpmn:InclusiveGateway') && property === 'incoming') {
+    return `${ getIndefiniteArticle(typeString) } <${ typeString }> with more than one incoming <Sequence Flow> is not supported by Camunda Fox`;
```

Comment:
Camunda Fox?

Reply:
Oh shit. Copy and paste error. 😓

Reply:
Fixed (cf. https://github.com/camunda/linting/pull/18/files#diff-bccb71f75065b812fa07bc4a28d88e5086d008607e0264420d03f02c46aa8da6R215).